front/app/containers/Admin/projects/edit/index.tsx

```diff
@@ -274,13 +274,7 @@ export class AdminProjectEdition extends PureComponent<
   };
 
   goBack = () => {
-    const backUrl =
-      this.state.goBackUrl &&
-      this.state.goBackUrl !== this.props.location.pathname
-        ? this.state.goBackUrl
-        : '/admin/projects';
-
-    clHistory.push(backUrl);
+    clHistory.push(`/admin/projects/`);
   };
```

Comment:
Simplifies the Go Back button so it takes you to the projects directory each time (the expected behavior).

Reply:
Isn't there a test that has to be adapted for this change? If not, shouldn't we add a test?

Reply:
For sure. I added a note in the description - I just wanted to wait until Iva's e2e tests are merged and then I could add a test there. I think she did this yesterday evening, so I can add this today.

Reply:
@IvaKop I'm a bit unsure in the refactored setup where I should add this test? Do you have a suggestion?

Reply:
Perhaps you can create a new test in the content builder test folder where you test the navigation between all the moving parts - edit project description, the content builder itself and the project?

Reply:
@IvaKop Ok, a general navigation test file sounds good :) Thanks!